service-catalog.spec

```diff
@@ -0,0 +1,150 @@
+#
+# This is a template package spec that will support Go builds following the OpenShift conventions.
+# It expects a set of standard env vars that define the Git version being built and can also handle
+# multi-architecture Linux builds. It has stubs for cross building.
+
+#debuginfo not supported with Go
+%global debug_package %{nil}
+
+# modifying the Go binaries breaks the DWARF debugging
+%global __os_install_post %{_rpmconfigdir}/brp-compress
+
+# %commit and %os_git_vars are intended to be set by tito custom builders provided
+# in the .tito/lib directory. The values in this spec file will not be kept up to date.
+%{!?commit: %global commit HEAD }
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+# os_git_vars needed to run hack scripts during rpm builds
+%{!?os_git_vars: %global os_git_vars OS_GIT_VERSION='' OS_GIT_COMMIT='' OS_GIT_MAJOR='' OS_GIT_MINOR='' OS_GIT_TREE_STATE='' }
+
+%if 0%{?skip_build}
+%global do_build 0
+%else
+%global do_build 1
+%endif
+%if 0%{?skip_prep}
+%global do_prep 0
+%else
+%global do_prep 1
+%endif
+
+%if 0%{?fedora} || 0%{?epel}
+%global need_redistributable_set 0
+%else
+# Due to library availability, redistributable builds only work on x86_64
+%ifarch x86_64
+%global need_redistributable_set 1
+%else
+%global need_redistributable_set 0
+%endif
+%endif
+%{!?make_redistributable: %global make_redistributable %{need_redistributable_set}}
+
+#
+# Customize from here.
+#
+
+%global golang_version 1.8.1
+%{!?version: %global version 0.0.1}
+%{!?release: %global release 1}
+%global package_name origin-service-catalog
+%global product_name Service Catalog
+%global import_path github.com/openshift/service-catalog
+
+Name:           %{package_name}
+Version:        %{version}
+Release:        %{release}%{?dist}
+Summary:        Consume services in Kubernetes using the Open Service Broker API
+License:        ASL 2.0
+URL:            https://%{import_path}
+
+Source0:        https://%{import_path}/archive/%{commit}/%{name}-%{version}.tar.gz
+BuildRequires:  golang >= %{golang_version}
+
+# If go_arches not defined fall through to implicit golang archs
+%if 0%{?go_arches:1}
+ExclusiveArch:  %{go_arches}
+%else
+ExclusiveArch:  x86_64 aarch64 ppc64le s390x
+%endif
+
+### AUTO-BUNDLED-GEN-ENTRY-POINT
+
+%description
+%{summary}
+
+%package svcat
+Summary: A domain-specific CLI tool to make interacting with Service Catalog easier
+
+%description svcat
+${summary}
+
+%prep
+%if 0%{do_prep}
+%setup -q
+%endif
+
+%build
+%if 0%{do_build}
+%if 0%{make_redistributable}
+# Create Binaries for all internally defined arches
+%{os_git_vars} hack/build-cross.sh
```

Comment:
Please always use make build-cross

Reply:
I didn't pull in the Makefile from origin. Are you sure this isn't workable?

Reply:
I don't think we should block on getting the build just-so in this PR, since, as I understand it, creation of the distgit repo is gated on having an RPM spec.

Is it cool if we whittle on the Makefile in a follow-up and just get the gears moving on the distgit repo for now?

Reply:
The makefile is automatically injected by the sync tool.  You used the sync tool right?

Reply:
Apparently it doesn't overwrite an already existing Makefile, so I didn't notice the tool generated one. But yes, I did use the tool.